app/services/katello/event_daemon/monitor.rb

```diff
@@ -18,13 +18,11 @@ def start
       end
 
       def check_services(error, status)
+        Katello::EventDaemon::Runner.cached_status(
+          refresh: true
```

Comment:
I changed `cached_status` so that `service_name` is optional as well.  If not provided it will just update all the statuses.